RELEASE_INFO-6.7.md

```diff
@@ -179,6 +179,10 @@ New extensible Twig blocks `layout_header_actions_language_widget_content_inner`
 The `context.token` variable is no longer available in twig rendering context to prevent potential security vulnerabilities. If you need to access the token, consider using alternative methods that do not expose it in the rendered HTML.
 Usually inside the Twig storefront there is no need to handle the context token manually, as it is handled automatically via the session handling in the Storefront.
 
+### The email validation supports IDN email addresses
```

Comment:
please rebase and put it under 6.7.6.0 headline, your PR won't make it to 6.7.5.0 anymore